### PR TITLE
first draft of section1 for data science pipelines

### PR DESCRIPTION
**Pull request by codrinbucur**
first draft of section1 for data science pipelines. please review!

### CODE REVIEW

**Review by rsriniva — APPROVED** [review]
Approving for now. Let's merge this

**Comment by rsriniva** [comment]
@mamurak - Please review Codrin's introduction to Pipelines lecture in this PR. The HTML rendering is broken because this PR was created from a fork. Please review raw asciidoc file in this PR

File: modules/chapter1/pages/section1.adoc [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+These activities are typically iterative and may involve going back and forth between different stages to refine the analysis and models. Data science pipelines can vary depending on the specific project and organization, but these general steps provide a framework for approaching data science tasks.
+
+
+== Data Science Pipelines in OpenShift AI
+
+From the perspective of a data scientist or ML engineer user experience, OpenShift AI provides several user interface capabilities trying to cover as many as possible from these data science activities. It includes the following
+
+* *Data Science Pipelines Main View* - End users can find the pipelines to whom they have access under the data science pipelines menu / main view
+
+image::../images/dsps-main.png[Data Science Pipelines Menu / Main View]
```
Comment (jramcast) [inline modules/chapter1/pages/section1.adoc]: Antora is not rendering the images cause the paths are incorrect. Remove the `../images` portion in this image and the remaining ones.



image



```suggestion
image::dsps-main.png[Data Science Pipelines Menu / Main View]
```

**Review by jramcast — COMMENTED** [review]
Good intro section! Make sure you fix image paths.

File: modules/chapter1/pages/section1.adoc [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+A data scientist can assign resources visually, including CPUs and GPUs, to each individual step in the pipeline.
+
+image::../images/elyra-pipeline-step-config-with-gpu.png[Configuring GPUs and other resources for an Elyra pipeline step]
+
+* *Kubeflow Pipelines* - Specialized data science pipelines engine which can translate an Elyra visual pipeline execution (or Kubeflow SDK calls) into a *Tekton* pipeline running in OpenShift.
+
+image::../images/elyra-pipeline-running.png[Starting an Elyra pipeline]
+image::../images/elyra-pipeline-job-started.png[Details of a started Elyra pipeline]
+
+* *OpenShift Pipelines (based on Tekton)* - It executes each step in the pipeline as an individual container / job and ensures that each containers gets allocated the resources (GPU, CPU, memory) configured/requested by data scientists in Elyra (or via Kubeflow SDK), as long as they are available in the OpenShift cluster.
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
* *OpenShift Pipelines (based on Tekton)* - It executes each step in the pipeline as an individual container and ensures that each container gets allocated the resources (GPU, CPU, memory) configured/requested by data scientists in Elyra (or via Kubeflow SDK), as long as they are available in the OpenShift cluster.
```

Technically, Tekton is based on pods, not jobs.

File: modules/chapter1/pages/section1.adoc [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+
+
+* *OpenShift GitOps with ArgoCD for AI/ML flows* - While not part of the OpenShift AI but rather as part of OpenShift, it is worth mentioning that once ML models are created, GitOps can be leveraged to push them to other OpenShift instances (including OpenShift Edge devices) for being hosted on model servers for inference and/or embedded in intelligent apps.
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
* *OpenShift GitOps with ArgoCD for automated model deployments* - While not part of the OpenShift AI but rather as part of OpenShift, it is worth mentioning that once ML models are created, GitOps can be leveraged to push them to other OpenShift instances (including OpenShift Edge devices) for being hosted on model servers for inference and/or embedded in intelligent apps.
```

**Review by mamurak — COMMENTED** [review]
Added two minor comments. Looks good to me overall.

File: modules/chapter1/pages/section1.adoc (hunk 1 above)
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::dsps-main.png[Data Science Pipelines Menu / Main View]
```

File: modules/chapter1/pages/section1.adoc [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+
+From the perspective of a data scientist or ML engineer user experience, OpenShift AI provides several user interface capabilities trying to cover as many as possible from these data science activities. It includes the following
+
+* *Data Science Pipelines Main View* - End users can find the pipelines to whom they have access under the data science pipelines menu / main view
+
+image::../images/dsps-main.png[Data Science Pipelines Menu / Main View]
+
+* *Data Science Pipelines Runs* - When looking for more details, end users explore the runs of their data science pipelines
+
+image::../images/dsps-runs.png[Data Science Pipelines Runs]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::dsps-runs.png[Data Science Pipelines Runs]
```

File: modules/chapter1/pages/section1.adoc [hunk 5]
```diff
@@ -1,3 +1,82 @@
...
+
+* *Data Science Pipelines view within a Data Science Project* - While within their data science project, end users can visualise the runs and other details of pipelines that ran within that project.
+
+image::../images/dsps-in-ds-project.png[Data Science Pipelines view within a Data Science Project]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::dsps-in-ds-project.png[Data Science Pipelines view within a Data Science Project]
```

File: modules/chapter1/pages/section1.adoc [hunk 6]
```diff
@@ -1,3 +1,82 @@
...
+
+Those data scientists and ML engineers with access to and knowledge of containers and OpenShift, can also look "under the covers" and check their pipeline runs direct in OpenShift:
+
+* *OpenShift Pipeline executing a Data Science Pipeline* - From the OpenShift Console, under Pipelines->PipelinesRuns one can see the underlying Tekton pipeline that runs in OCP, once a data scientist started a Data Science Pipeline in OpenShift AI
+
+image::../images/dsp-run-in-ocp-pipelines.png[OpenShift Pipeline executing a Data Science Pipeline]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::dsp-run-in-ocp-pipelines.png[OpenShift Pipeline executing a Data Science Pipeline]
```

File: modules/chapter1/pages/section1.adoc [hunk 7]
```diff
@@ -1,3 +1,82 @@
...
+
+
+
+== Data Science Pipelines Technologies
+
+At a high level, here are the technologies used to implement Data Science Pipelines in OpenShift AI:
+
+* *Elyra Pipelines* - A *JupyterLab* extension, which provides a visual editor for creating data science pipelines based on Jupyter notebooks as well as Python or R scripts. User can drag and drop code and create visually such pipelines
+
+image::../images/elyra-pipeline-offline-scoring.png[An Offline Scoring pipeline in Elyra]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::elyra-pipeline-offline-scoring.png[An Offline Scoring pipeline in Elyra]
```

File: modules/chapter1/pages/section1.adoc [hunk 8]
```diff
@@ -1,3 +1,82 @@
...
+image::../images/elyra-pipeline-model-training.png[A Model Training pipeline in Elyra]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::elyra-pipeline-model-training.png[A Model Training pipeline in Elyra]
```

File: modules/chapter1/pages/section1.adoc [hunk 9]
```diff
@@ -1,3 +1,82 @@
...
+
+A data scientist can assign resources visually, including CPUs and GPUs, to each individual step in the pipeline.
+
+image::../images/elyra-pipeline-step-config-with-gpu.png[Configuring GPUs and other resources for an Elyra pipeline step]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::elyra-pipeline-step-config-with-gpu.png[Configuring GPUs and other resources for an Elyra pipeline step]
```

File: modules/chapter1/pages/section1.adoc [hunk 10]
```diff
@@ -1,3 +1,82 @@
...
+
+* *Kubeflow Pipelines* - Specialized data science pipelines engine which can translate an Elyra visual pipeline execution (or Kubeflow SDK calls) into a *Tekton* pipeline running in OpenShift.
+
+image::../images/elyra-pipeline-running.png[Starting an Elyra pipeline]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::elyra-pipeline-running.png[Starting an Elyra pipeline]
```

File: modules/chapter1/pages/section1.adoc [hunk 11]
```diff
@@ -1,3 +1,82 @@
...
+image::../images/elyra-pipeline-job-started.png[Details of a started Elyra pipeline]
```
Comment (mamurak) [inline modules/chapter1/pages/section1.adoc]:
```suggestion
image::elyra-pipeline-job-started.png[Details of a started Elyra pipeline]
```

**Review by mamurak — CHANGES_REQUESTED** [review]
Highlighted necessary changes for image rendering.

**Comment by codrinbucur** [comment]
Committed all suggestions from Max and Jaime

**Pull request merged**